.github/workflows/tag_action.yml

```diff
@@ -0,0 +1,35 @@
+name: Report tag to Notion
+
+on:
+  push:
+    tags:
+      - rc*
+env:
+  NOTION_DATABASE_ID: e154131912ef432aa13a80346d0f4b66
+  WEBHOOK_URL: https://hook.eu2.make.com/sg1xhhlvnp4314llgde92csqgjrpm42e
+
+jobs:
+  report_tag_to_notion:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

Comment:
This is at @v4 now (or even @v4.1.7 if you want).

The same in the other action .yml

Reply:
fixed